vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/main/java/com/vaadin/flow/component/combobox/test/ClearValuePage.java

```diff
@@ -15,73 +15,99 @@
  */
 package com.vaadin.flow.component.combobox.test;
 
+import java.util.List;
+import java.util.Arrays;
+
+import com.vaadin.flow.component.Component;
 import com.vaadin.flow.component.combobox.ComboBox;
 import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.H2;
 import com.vaadin.flow.component.html.NativeButton;
 import com.vaadin.flow.component.html.Paragraph;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
 import com.vaadin.flow.router.Route;
 
 @Route("vaadin-combo-box/clear-value")
 public class ClearValuePage extends Div {
     static final String INITIAL_VALUE = "two";
+    static final List<String> ITEMS = Arrays.asList("one", "two", "three");
```

Comment:
**note:** I had to use `Arrays.asList` instead `List.of` since Java-SE 1.8 doesn't support the latter.